src/CMakeLists.txt

```diff
@@ -10,10 +10,12 @@ endif()
 add_subdirectory(CubicInterpolation)
 add_subdirectory(detail)
 
-target_link_libraries(CubicInterpolation
-    CONAN_PKG::eigen
-    CONAN_PKG::boost
-    )
+target_link_libraries(CubicInterpolation PUBLIC
+    Eigen3::Eigen
+    Boost::boost
```

Comment:
I can't find a lib called `boost`. This may cause the error in travis.
I think this must be something like 
```
Boost::math                                                                                                                                                                                                                           
Boost::serialization                             
Boost::filesystem
```
to make travis run

Reply:
Boost::boost should be the header only libraries needed for the includes. But yes, the other ones are probably missing.

Reply:
Ah okay that's something I missed. I personally find it much nicer than `include_directories(${Boost_INCLUDE_DIRS})`